app/tests/system/nightwatch-config.js

```diff
@@ -32,7 +32,12 @@ module.exports = {
         'default': {
             'globals': {
                 'waitForConditionTimeout': 60000,
-                'waitForConditionPollInterval': 500
+                'waitForConditionPollInterval': 500,
+                // WEBVIEWS GO HERE. Are set dynamically on test run
+                "contexts": {
+                    'NO_INTERNET': '',
```

Comment:
I think it would be more clear if the default value here was `null`. These won't be strings when we set them so it is a bit misleading to default them to an empty string here.